bin/format

```diff
@@ -26,7 +26,7 @@ npm exec prettier -- \
   --config "$PROJECT_ROOT/.prettierrc" \
   --ignore-path "$PROJECT_ROOT/.prettierignore" \
   --write \
-  js/collaborative-editor test | grep -v "unchanged" || {
+  "**/*.{js,jsx,ts,tsx}" | grep -v "unchanged" || {
```

Comment:
I think this should just be `.` or `js test`. Prettier has it's own config about what files to handle, so we could even run the from the root.